lib/awspec/helper/finder/cloudtrail.rb

```diff
@@ -15,6 +15,12 @@ def get_trail_status(id)
         cloudtrail_client.get_trail_status(name: id)
       end
 
+      def get_trail_tags(arn)
+        cloudtrail_client.list_tags(
+          resource_id_list: [arn]
+        )[:resource_tag_list].first[:tags_list]
```

Comment:
Does `list_tags` always return only one?
If `list_tag` can return more than one value, you can use [Array#single_resource](https://github.com/k1LoW/awspec/blob/3e8483d8bba07337f8f686c2d61976517cc1d125/lib/awspec/ext/array.rb#L2)

Reply:
thanks, let me look into it. Last I remembered it seemed to always return an array with a single element

Reply:
Ok, given only a single `arn` in the `resource_id_list`, `list_tags` should always return only a hash with a single element array in the `:resource_tag_list` key

Reply:
https://docs.aws.amazon.com/cli/latest/reference/cloudtrail/list-tags.html